package.json

```diff
@@ -12,7 +12,7 @@
     "gyp"
   ],
   "version": "9.3.1",
-  "installVersion": 10,
```

Comment:
A question for people with more experience in `node-gyp`: Since a PR that changed `installVersion` from 9 to 10 landed yesterday in https://github.com/nodejs/node-gyp/commit/aaa117c514430aa2c1e568b95df1b6ed1c1fd3b6 do I need to increase it to 11 now, or not, because there was no release with `installVersion: 10`?

Reply:
> A question for people with more experience in `node-gyp`

I'm not more experienced, but my 2 cents: it's probably unnecessary for end users (since as you stated, no release went out), but AFAIK there's no harm in bumping it again, and it will prevent a situation where anyone who tested `main` (so folks working on `node-gyp` like you and I, or maybe even someone some where consuming `node-gyp@main` for whatever reason) and got their cache populated for `installVersion: 10` won't end up in a broken situation.

So to me bumping it to 11 seems reasonable to prevent any problems.